### PR TITLE
Dev: less potential issue with console

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: config-default rootdir to realpath(dirname(dirname(dirname(__FILE__))))
Dev: with set a good rootdir, all other are better set
Dev: allow usage of configdir : i NEVER use this settings

### CODE REVIEW

File: application/config/config-defaults.php [hunk 1]
```diff
@@ -38,11 +38,13 @@
...
-$config['rootdir'] = getcwd(); //dirname(__FILE__); // This is the physical disk location for your limesurvey installation. Normally you don't have to touch this setting.
-// If you use IIS then you MUST enter the complete rootdir e.g. : $rootDir="C:\Inetpub\wwwroot\limesurvey"!
-// Some IIS installations also require to use forward slashes instead of backslashes, e.g.  $rootDir="C:/Inetpub/wwwroot/limesurvey"!
-// If you use OS/2 this must be the complete rootdir with FORWARD slashes e.g.: $rootDir="c:/limesurvey";!
-
+/**
+* rootdir
+* Physical disk location for your limesurvey installation
+* Normally you don't have to touch this setting
+* Usage of dirname like intenal.php
```
Comment (tammoterhark) [inline application/config/config-defaults.php:45]: intenal or internal?

**Review by olleharstedt — CHANGES_REQUESTED** [review]
Needs to pull and resolve conflicts again, it seems.

**Comment by c-schmitz** [comment]
@Shnoulle You still want to merge this?

**Comment by Shnoulle** [comment]
Yes, but need fix (and test)

I set to draft , near one year old request

**Comment by c-schmitz** [comment]
Same here: Either we finish this or we close it.

**Comment by Shnoulle** [comment]
I close , i do another pull request

**Pull request closed without merging**